packages/catalog-model/src/kinds/GroupEntityV1alpha1.test.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { EntityPolicy } from '../types';
+import {
+  GroupEntityV1alpha1,
+  GroupEntityV1alpha1Policy,
+} from './GroupEntityV1alpha1';
+
+describe('GroupV1alpha1Policy', () => {
+  let entity: GroupEntityV1alpha1;
+  let policy: EntityPolicy;
+
+  beforeEach(() => {
+    entity = {
+      apiVersion: 'backstage.io/v1alpha1',
+      kind: 'Group',
+      metadata: {
+        name: 'doe-squad',
+        title: 'Doe Squad',
+        description: 'A squad for John and Jane',
+      },
+      spec: {
+        type: 'squad',
+        parent: 'group-a',
+        ancestors: ['group-a', 'global-synergies', 'acme-corp'],
+        children: ['child-a', 'child-b'],
+        descendents: ['desc-a', 'desc-b'],
```

Comment:
sorry, but it should be `descendants`! should be possible to just search-and-replace everywhere

Reply:
oops, that shows me as well for not spell checking.